src/main/scala/ch/openolitor/buchhaltung/rechnungsexport/iso20022/Pain008_001_02_Export.scala

```diff
@@ -52,10 +52,16 @@ class Pain008_001_02_Export extends LazyLogging {
                     CustomerDirectDebitInitiationV02(grpHeader, Seq(pmtInf))
                   ), "Document", defineNamespaceBinding()).toString()
             }
-          case _ => ""
+          case _ => {
+            logger.debug("Invalid payment information while creating a pain008_001_02 file")
```

Comment:
In PROD debug message will not be shown in the logs. As this is an error, I would go for this level.

Reply:
done

Reply:
:+1: